indico/modules/events/abstracts/operations.py

```diff
@@ -367,13 +367,15 @@ def update_abstract_review(review, review_data, questions_data):
         field_name = 'question_{}'.format(question.id)
         rating = question.get_review_rating(review, allow_create=True)
         old_value = rating.value
-        rating.value = int(questions_data[field_name])
+        rating.value = questions_data[field_name]
         if old_value != rating.value:
-            changes[field_name] = (old_value, rating.value)
+            changes[field_name] = (question.field.get_friendly_value(old_value),
+                                   question.field.get_friendly_value(rating.value))
             log_fields[field_name] = {
                 'title': question.text,
                 'type': 'number'
```

Comment:
this needs to be set according to the field type as it determines how the diff is displayed in the log message